src/types/indexeddb.js

```diff
@@ -97,7 +97,12 @@ export interface IDBObjectStore<K, V> {
   delete(key: K): IDBRequest<void>;
```

Comment:
Note: I don't use these changes after all, because we use the library `idb` now, but I left them because it could still be useful for other cases in the future.